### PR TITLE
paste from register

### PR DESCRIPTION
**Pull request by alibby**

### CODE REVIEW

File: _posts/2018-11-01-paste-from-register-in-command.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+me. There's a way to paste the contents of a vim register in the vim command
+window.  Using the `<C-r>` in command mode will interpret the next character(s)
+as a register and insert the contents of the register into the vim command!
+
+A common use for this is to search for an entire string literal using `Ggrep`:
+
+
+{% highlight plain %}
+  def is_active?
+    status == 'slaes active'
```
Comment (cbartlett) [inline _posts/2018-11-01-paste-from-register-in-command.md]: typo slaes?

File: _posts/2018-11-01-paste-from-register-in-command.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: paste from register in command
+date: 2018 Nov 01 07:09:25
```
Comment (cbartlett) [inline _posts/2018-11-01-paste-from-register-in-command.md:4]: Does this need to be YYYY-MM-DD HH:MM:SS or does this work?

**Review by cbartlett — CHANGES_REQUESTED** [review]
two tiny things

File: _posts/2018-11-01-paste-from-register-in-command.md (hunk 1 above)
Comment (alibby) [reply _posts/2018-11-01-paste-from-register-in-command.md]: Thanks.

File: _posts/2018-11-01-paste-from-register-in-command.md (hunk 2 above)
Comment (alibby) [reply _posts/2018-11-01-paste-from-register-in-command.md:4]: Works in preview mode.

**Comment by alibby** [comment]
Okay, I think I got it.  I didn't change the date because it works locally, if you think it needs to be changed press on me.

**Pull request merged**